### ISSUE TITLE
Different keyboards to search subreddit list

### ISSUE THREAD

**Issue by matty1086** [ccrama/Slide#1950]
Using the latest Google keyboard (v5.1) when i pull out the sidebar to search for subreddits there, it brings up a keyboard with a number row, which does not allow swipe input. In my experience, the number row pops up when the keyboard thinks the input field is a password field.

If i long press on the subreddit name to bring up the list, instead of number row, it shows the auto complete bar and does allow for swipe input.

Possible to change the input type on the sidebar input to match the input type when long pressing subreddit name so that swipe works?

Check out the following screenshots:

Sidebar - no swipe input allowed:
![screenshot_20160718-170828](https://cloud.githubusercontent.com/assets/17831198/17007366/f38faa04-4e9c-11e6-89ca-4598ca42610f.png)

Long press subreddit - swipe input allowed:
![screenshot_20160718-170846](https://cloud.githubusercontent.com/assets/17831198/17007372/fed9b5e4-4e9c-11e6-8778-8ea54521a605.png)

Slide version: 5.4 Alpha 2
Android version: N DP4

**Comment by Nxt3** [comment]
This is also Nougat specific. :p I'm sorry.

**Comment by matty1086** [comment]
Have you by chance looked at this to see what is causing this (and the issue in the other N specific bug I posted) in the code?
It would be great if these are N specific issues to get them reported while the release is still in preview on the Android bug tracker. I'd love to do it, but without knowing the code behind, I'm afraid I wouldn't be much use in a useful bug report for Google to look at. I think this would be the better route to go rather than hoping that Google fixes something that they may or may not think (or know) is an issue.

I totally understand not wanting to put a bunch of effort towards a preview release (that's what I get when I signed up for previews), but I'm assuming once N is officially released later this year to the masses, that there will be a push to ensure functionality in Slide is compatible with N?
